src/components/molecules/PluginEditor/hooks.ts

```diff
@@ -69,6 +70,11 @@ export default () => {
     },
   ];
 
+  const handleFileDownload = (body?: string, filename?: string) => {
+    if (!body) return;
+    fileDownload(body, filename ?? "undefined.js");
```

Comment:
Why not making filename required?
What do you think?
If not, is `undefined.js` is the best name?

Reply:
Its not required because the variables we pass into this function come from `string | undefined` type. 
If you dislike the fallback name to be `undefined.js`, please suggest a better name and I'll update it.